frontend/app/views/joiner/thankyou.scala.html

```diff
@@ -24,6 +26,17 @@
     }
 }
 
+@getStarted(content: Html) = {
```

Comment:
Can the case matching just go in side `page-section__content`? as nothing really changes here? Personal preference but not a huge fan of the inline fragments in these cases.